docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "2"
+services:
+  redis:
+    container_name: redis
+    image: redis
+    ports:
+      - 6379:6379
```

Comment:
this opens up the various services on the developers computer with no security - I _think_ this can specify the ip to bind to as well, ie

`127.0.0.1:6379:6379`

Reply:
good point, I'll add the address.